### DISCUSSION TITLE
Add support for XLIFF export/import

### DISCUSSION THREAD

**Discussion by yarick2000** [directus/directus#4658] [Feature Requests]
It will be nice to have support for exporting translations into XLIFF format as well as importing them back. There are organizations that outsourcing translation work to outside companies and using special services for that. One of the popular formats for uploading translations to such services is XLIFF.

**Comment by rijkvanzanten** [comment]
Like it!

Wondering if we can introduce it as a feature under the existing `export` sidebar drawer detail, which only shows up when you have a translations field in the current collection context. Likewise, the import of XLIFF should be handled the same as we'll handle regular data imports (ref https://github.com/directus/directus/issues/2927)

**Reply by yarick2000** [reply]
Yes, this is implemented exactly as you've explained. Regarding to import - I'm planning to make it as a separate PR.

**Comment by yarick2000** [comment]
@rijkvanzanten , I've finished to work on export part. It can be reviewed here: https://github.com/directus/directus/pull/4697.
Also started to work on import, will provide with screenshots soon

**Reply by yarick2000** [reply]
@rijkvanzanten , here are the UI screenshots for import:
1. Language manually selected (there are multiple translation fields in collection)
![image](https://user-images.githubusercontent.com/13006578/113589424-54647180-95ff-11eb-9658-88eea21cfe14.png)
2. Language taken from imported file
![image](https://user-images.githubusercontent.com/13006578/113589509-6cd48c00-95ff-11eb-964b-20f7ad990069.png)
3.  Language manually selected (only one translation field in collection)
![image](https://user-images.githubusercontent.com/13006578/113589829-db194e80-95ff-11eb-82fd-dd031ac50857.png)

**Comment by yarick2000** [comment]
The new PR for both export and import has been created here:
https://github.com/directus/directus/pull/5040

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
